ui/src/views/Secrets/ViewSecrets/index.jsx

```diff
@@ -65,19 +98,33 @@ export default class ViewSecrets extends Component {
     });
   };
 
+  handleSecretSearchChange = ({ target: { value } }) => {
+    this.setState({ value });
+  };
+
   render() {
     const {
       classes,
       description,
       data: { loading, error, secrets },
     } = this.props;
+    const { value } = this.state;
 
     return (
       <Dashboard
         title="Secrets"
-        helpView={<HelpView description={description} />}>
+        helpView={<HelpView description={description} />}
+        search={
+          <Search
+            disabled={loading}
+            onSubmit={this.handleSecretSearchSubmit}
+            onChange={this.handleSecretSearchChange}
+            value={value}
```

Comment:
No need to have `value` and `onChange`. You can grab the value directly from `onSubmit`.

Reply:
Alright